Mock/postMock.js

```diff
@@ -1,19 +1,17 @@
 export default {
   success: {
     path: 'tests/create/animals',
-    body: { specimen: 'unkown', name: 'None' },
-    newBody: { specimen: 'Shark', name: 'Little Shark' },
+    body: { name: 'Bob the test Bear', specimen: 'Bear' },
     response: {
       ok: true,
-      request: 'GET',
-      status: 200,
+      request: 'POST',
+      status: 201,
```

Comment:
It would be nice to use constants for the methods and the code statuses

Reply:
I will add a card to refactor the mock files as to not have a lot of changes on this pr :+1: